hack/build-antrea-linux-all.sh

```diff
@@ -118,6 +128,9 @@ GO_VERSION=$(head -n 1 build/images/deps/go-version)
 
 BUILD_TAG=$(build/images/build-tag.sh)
 echo "BUILD_TAG: $BUILD_TAG"
+if [ "$CUSTOM_BUILD_TAG" != "" ]; then
```

Comment:
The `build-antrea-linux-all.sh` script is also used in other files. Given that `build_tag.sh` is no longer used during image building, should they add extra parameters to specify the tag? Personally, I prefer to continue using build_tag.sh to generate tags when a specific tag is not specified in the Makefile. This can better maintain compatibility with the previous code without modifications.

Reply:
yes it still uses the build-tag.sh script if no CUSTOM}_BUILD_TAG is specified.

Reply:
I have a question: Why was `build-tag.sh` removed from the Makefile and added to build.sh? This way, for all make commands without a specific tag, build-tag.sh won't be executed.

Reply:
no that's not the case, this line is added to the versioning.mk file, so that if no custom tag is provided, automatically the build-tag.sh script will be executed.

Reply:
I see, I missed that file. Thanks for the explanation.